packages/expo-env-info/package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "expo-env-info",
   "version": "1.0.2",
-  "main": "build",
+  "exports": {
+    ".": "./build/index.js"
+  },
```

Comment:
Is the lib used anywhere as an imported module? If not maybe we could drop this?

Reply:
We don't import it anywhere, so I'm fine with dropping it.